exporting/pubsub/metadata.yaml

```diff
@@ -0,0 +1,156 @@
+          required: false
+        - name: 'send hosts matching'
+          default_value: 'localhost *'
+          description: |
+            Hosts filter. Determines which hosts will be sent to the external database. The syntax is [simple patterns](https://github.com/netdata/netdata/tree/master/libnetdata/simple_pattern#simple-patterns).
+          required: false
+          detailed_description: |
+            Includes one or more space separated patterns, using * as wildcard (any number of times within each pattern).
+            The patterns are checked against the hostname (the localhost is always checked as localhost), allowing us to
+            filter which hosts will be sent to the external database when this Netdata is a central Netdata aggregating multiple hosts.
+
+            A pattern starting with `!` gives a negative match. So to match all hosts named `*db*` except hosts containing `*child*`,
+            use `!*child* *db*` (so, the order is important: the first pattern matching the hostname will be used - positive or negative).
+        - name: 'send charts matching'
+          default_value: '*'
+          description: |
+            One or more space separated patterns (use * as wildcard) checked against both chart id and chart name.
+          required: false
+          detailed_description: |
+            A pattern starting with ! gives a negative match. So to match all charts named apps.* except charts ending in *reads,
+            use !*reads apps.* (so, the order is important: the first pattern matching the chart id or the chart name will be used,
+            positive or negative). There is also a URL parameter filter that can be used while querying allmetrics. The URL parameter
+            has a higher priority than the configuration option.
+        - name: 'send names instead of ids'
+          default_value: ''
+          description: 'Controls the metric names Netdata should send to the external database (yes|no).'
+          required: false
+          detailed_description: |
+            Netdata supports names and IDs for charts and dimensions. Usually IDs are unique identifiers as read by the system and names
+            are human friendly labels (also unique). Most charts and metrics have the same ID and name, but in several cases they are
+            different : disks with device-mapper, interrupts, QoS classes, statsd synthetic charts, etc.
+        - name: 'send configured labels'
+          default_value: ''
+          description: 'Controls if host labels defined in the `[host labels]` section in `netdata.conf` should be sent to the external database (yes|no).'
+          required: false
+        - name: 'send automatic labels'
+          default_value: ''
+          description: 'Controls if automatically created labels, like `_os_name` or `_architecture` should be sent to the external database (yes|no).'
+          required: false
```

Comment:
These are not described in the exporter configuration reference document. So, we are treating these as custom options relevant to specific integrations.

Reply:
If there's information available somewhere or if you know please add it, otherwise we can take care of this post release. I could not find anything in learn about these fields.

Reply:
Credentials and Project ID are described [here](https://cloud.google.com/pubsub/docs/authentication) and we have topics [here](https://cloud.google.com/pubsub/docs/create-topic)..

I will proceed with reviews and plugins I have in my backlog and at the end of the day  I will return and push commits here.